app/models/concerns/event_concern.rb

```diff
@@ -8,9 +8,9 @@ module EventConcern
   IGNORE_RAISE_EVENT_ATTRIBUTES_LIST = %i[availability_status availability_status_error].freeze
 
   IGNORE_RAISE_EVENT_LIST = {
```

Comment:
Pls add a TODO comment

Reply:
I'm not sure what the `TODO` would be? I don't see us re-enabling this since subswatch needs it.

Reply:
I'm wondering it could be something like 
```
# TODO: IGNORE_RAISE_EVENT_ATTRIBUTES_LIST will be added later
```

Reply:
added